### ISSUE TITLE
[EPIC] Form 10-7959c CHAMPVA Other Health Insurance (OHI) certification

### ISSUE THREAD

**Issue by marywang2** [department-of-veterans-affairs/va.gov-team#77855]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to start collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

**Note:** If you’re already partnering with us on an initiative, you don't need to fill out this request form. If you don’t have access to your initiative’s CAIA epic, post a message in the [#sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-acessibility-ia) slack channel and tag `@Terry Nichols` and `@Lily Strelich`.

## About your team

- **Team name**: IVC Forms
- **OCTO product owner:** Premal Shah
- **Product name:** 10-7959c, Other Health Insurance (OHI) Certification 
- **Product manager:** Bo Altes
- **Slack channel:** ivc-forms
- **Dedicated content writer on your team (if you have one):** n/a
- **Dedicated a11y specialist on your team (if you have one):** Rachael Penfil
 
### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [x] Digitizing a new form
- [ ] Creating a new digital tool
- [ ] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [ ] Something else:  `insert description`

### What's the nature of your initiative and desired outcomes?

Our team will be digitizing existing forms for the CHAMPVA program and Foreign Medical Program, among other Veteran and Family Member programs. We would like CAIA's support in reviewing content changes or changes wtih accessibility impacts.

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [ ] Yes
- [x] No
- [ ] I’m not sure

### Where are you at in your timeline?
Early user flows

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [x] Design intent 
- [x] Midpoint review 
- [ ] Staging 
- [ ] None. This initiative isn’t going through the collaboration cycle.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide your ticket number and link:
- [Ticket Number](https://app.zenhub.com/workspaces/ivc-forms-652da2d3f0ae4c0016bfb109/issues/gh/department-of-veterans-affairs/va.gov-team/74306)

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.
- 
-[Project initiative](https://github.com/department-of-veterans-affairs/va.gov-team/issues/83203)
- [Figma early flows](https://www.figma.com/file/sSPoS7R9HE6ZlxRESRy2ms/10-7959c-Form?type=design&node-id=37-1574&mode=design&t=IEOvNC6wt7cCeLlc-0)
- [Figma Wireframes](https://www.figma.com/file/sSPoS7R9HE6ZlxRESRy2ms/10-7959c-Form?type=design&node-id=190-4317&mode=design&t=kyn5exacq0u22pCq-0)

## Next steps
- [x] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Terry Nichols`. We’ll reply within 24 hours to acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [ ] Review the remaining next steps in the **VFS team tasks** section here.

## Tasks

<br>

```[tasklist]
### VFS team tasks
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well.<br> [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work. <br> [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team. <br> [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next. <br> [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
- [ ] **If you are requesting content support**, please share your content source of truth (preferably as a SharePoint document).
```

```[tasklist]
### CAIA a11y tasks
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/80754
```

**Comment by aliyahblackmore** [comment]
@strelichl tracking updates here on the 10-7959c (OHI) form.  

Jamie confirmed yesterday that this is the [Figma file](https://www.figma.com/design/sSPoS7R9HE6ZlxRESRy2ms/10-7959c-Form?node-id=402-30610&t=Fufq2VAv2QpifSYD-0) that I should work form. In section "Working file for CAIA comments_4/22/24" 

- I just wrapped up the content feedback/questions. I made some slight content changes to earlier screens (like the intro and signer section, and first health insurance screens) based on your more recent replies to questions.
- And there are a few outstanding questions throughout
  - The questions/convo tracked in Slack around POA, applicants applying on their own who are 18 or younger, Medicare and pharmacy benefits.
  - Some questions and flags about differences between the PDF and the online form
  - A few flags where things differentiate slightly from the design system.
  - A question about the free text field you all included
 - The product team scheduled their Midpoint review for 5/14 (scheduled yesterday). The team didn't share this with me, so I continued to wrap up my content feedback today. And let Jamie know that if they aren't able update their designs with all of the content adjustments ahead of sharing the artifacts, note for governance that there are still pieces of content feedback that they'll need to add to the designs.
 - For any outstanding questions that still need stakeholder confirmation, we'll need to revisit that content before they go to testing. But I've made adjustments based on the designs as they are right now.

**Comment by kristinoletmuskat** [comment]
Here is the IA spec for this work: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/information-architecture/ia-design-docs/champva-ohi-7959c.md

**Comment by laurwill** [comment]
Noting I updated the name of this ticket to make clear that this is for the OHI cert form

**Comment by marywang2** [comment]
Hi @laurwill @strelichl @aliyahblackmore! We would like to schedule staging review for OHI. Would you like to take another content review pass before we schedule it? Thank you! cc @jamiefiore

**Comment by marywang2** [comment]
Also wanted to note that the react widget ID is: form107959c. Please let me know if I should open a new ticket for the content creation or if it is fine to track here. @laurwill @strelichl

**Comment by aliyahblackmore** [comment]
Update here: 

As requested, I reviewed the confirmation screen and email content. Comments/adjustments are in Figma. 
:link:[OHI link](https://www.figma.com/design/sSPoS7R9HE6ZlxRESRy2ms/Other-Health-Insurance-Certification-(10-7959c)-Form?node-id=2680-43840&t=xXKJTZiCa6Mi4ioA-4)

And as we chatted earlier today, I'll hold on a second content review of the full OHI flow because you all mentioned that you got new information from your VA partners yesterday and may make further design adjustments.

**Comment by aliyahblackmore** [comment]
**Update here** 

Mary and Jamie joined office hours to talk about FMP reg and other emails. 

We confirmed that we'll move forward with making the CAIA content adjustments to the confirmation emails now. The adjustments across 1010D/OHi/CHAMPVA claim, where applicable, will align. And when CAIA has standard VA notify content, we'll circle back if there are any future adjustments that need to happen.

- We confirmed on the call that we'll move the contact information up under "What to expect next" and remove the header "How to contact us about your application."

- And we confirmed that we'll use the header "What to do if you need to submit supporting documents" for the supporting documents" information, where applicable

**Comment by aliyahblackmore** [comment]
**Update here after 12/12 IVC forms design/content sync:** 

- The product team wants to get to this before CHAMPVA claims. Jamie shared that they made some larger updates to the flow after midpoint.

- I let them know that we can do another content review before staging. They can let us know when the designs are ready.

**Comment by aliyahblackmore** [comment]
**Updates here:** 

- Jamie shared the [updated OHI designs](https://www.figma.com/design/sSPoS7R9HE6ZlxRESRy2ms/Other-Health-Insurance-Certification-(10-7959c)-Form?node-id=3049-43445&p=f&t=Psu3ypWbUD1GVXUl-0) today. They made a few updates to the Medicare screens and upload screens.
- They are going to staging on 1/14. 
- I'll start a review this week and the team will likely make adjustments after staging. 
- I let them know that they can flag in their staging review ticket that there are a few areas that they'll plan to update after staging.
- And I'll talk more with the team on Thursday about the entry points for the OHI form. This is something we'd need to talk more about bore MVP launch.
   - They shared that they only want the OHI to be a standalone form.
   - If someone learns about or knows that there's an online version of the OHI and they complete their OHI before they submit an online 1010D, how does the 1010D account for this? The online 1010D can let them know that they can submit the PDF version or complete the online version.

**Comment by aliyahblackmore** [comment]
**Update here:** 

- I'll continue my review of the MVP OHI screens tomorrow. I left off at the Medicare Part A screen. 
- The teams re-designs raise a few questions that we might need to consider and revisit before their MVP launch [will update this list with more sections]:
  - The Medicare coverage screen and radio button options - [link to screen](https://www.figma.com/design/sSPoS7R9HE6ZlxRESRy2ms/Other-Health-Insurance-Certification-(10-7959c)-Form?node-id=3744-170768&t=J8L2yDD4070k6xpP-4)
  - The Medicare coverage upload screen - [link to screen](https://www.figma.com/design/sSPoS7R9HE6ZlxRESRy2ms/Other-Health-Insurance-Certification-(10-7959c)-Form?node-id=3744-170663&t=J8L2yDD4070k6xpP-4)

**Comment by mnorthuis** [comment]
Updated IA spec for the following:
- Updated URL to reflect action word change in H1 ("file" vs "submit") per content recommendation above  
- Updated entry points to indicate which entry point is required for launch per the [launch blocking staging review issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/101005)
- Added entry point for left nav, can be added after launch
- Modified approach to best bets , to be handled after launch

[Final IA](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/information-architecture/ia-design-docs/champva-ohi-7959c.md)

**Comment by coforma-jamie** [comment]
Spoke with @jamiefiore, the accessibility work on this ticket is done.

> [!NOTE]
> ## Accessibility Digital Experience (DE) Ticket Status Update
> Accessibility DE is OCTO’s new accessibility team. We’re working on a new intake process. In the meanwhile:
> - This ticket has been moved to validate and will be closed on the Accessibility DE pipeline (only).
> - This ticket will remain “open” until the work from content and IA is completed.
> 
> If you need additional accessibility support, contact the Accessibility DE team on [#accessibility-help](https://dsva.slack.com/archives/C8E985R32).
> **Thank you for working with the accessibility team!**
